### PR TITLE
Policy Unit Testing

### PR DESCRIPTION
**Pull request by v-akarke**
Added Unit Testing for :
- Get-EntraMSPermissionGrantPolicy
- New-EntraMSNamedLocationPolicy
- New-EntraMSPermissionGrantPolicy 
- Set-EntraMSPermissionGrantPolicy
- Add-EntraMSLifecyclePolicyGroup
- New-EntraMSGroupLifecyclePolicy
- Set-EntraMSGroupLifecyclePolicy
- Get-EntraMSAuthorizationPolicy
- Set-EntraMSAuthorizationPolicy
- Get-EntraMSServicePrincipalDelegatedPermissionClassification
- Get-EntraBetaMSPermissionGrantPolicy

### CODE REVIEW

File: test/module/Entra/Get-EntraAuthorizationPolicy.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                "Id"                                        = "authorizationPolicy"
+                "Parameters"                                = $args
+            }
+        )
+    }
+
+    Mock -CommandName Get-MgPolicyAuthorizationPolicy -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "Get-EntraAuthorizationPolicy" {
```
Comment (KenitoInc) [inline test/module/Entra/Get-EntraAuthorizationPolicy.Tests.ps1:36]: Add test for parameter `Id`

File: test/module/Entra/Add-EntraLifecyclePolicyGroup.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                "AdditionalProperties" = "{[@odata.context, https://graph.microsoft.com/v1.0/$metadata#Edm.Boolean]}"
+                "Parameters"           = $args
+            }
+        )
+    }
+
+    Mock -CommandName Add-MgGroupToLifecyclePolicy -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "Add-EntraLifecyclePolicyGroup " {
```
Comment (KenitoInc) [inline test/module/Entra/Add-EntraLifecyclePolicyGroup.Tests.ps1:23]:
```suggestion
Describe "Add-EntraLifecyclePolicyGroup" {
```

File: test/module/Entra/New-EntraGroupLifecyclePolicy.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                "ManagedGroupTypes"           = "Selected"
+                "Parameters"                  = $args
+            }
+        )
+    }
+
+    Mock -CommandName New-MgGroupLifecyclePolicy -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "New-EntraGroupLifecyclePolicy " {
```
Comment (KenitoInc) [inline test/module/Entra/New-EntraGroupLifecyclePolicy.Tests.ps1:25]:
```suggestion
Describe "New-EntraGroupLifecyclePolicy" {
```

File: test/module/Entra/New-EntraNamedLocationPolicy.Tests.ps1 [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+                }
+                "Parameters"           = $args
+            }
+        )
+    }
+
+    Mock -CommandName New-MgIdentityConditionalAccessNamedLocation -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "New-EntraNamedLocationPolicy " {
```
Comment (KenitoInc) [inline test/module/Entra/New-EntraNamedLocationPolicy.Tests.ps1:35]:
```suggestion
Describe "New-EntraNamedLocationPolicy" {
```

File: test/module/Entra/Set-EntraGroupLifecyclePolicy.Tests.ps1 [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                "ManagedGroupTypes"           = "All"
+                "Parameters"                  = $args
+            }
+        )
+    }
+
+    Mock -CommandName Update-MgGroupLifecyclePolicy -MockWith $scriptblock -ModuleName Microsoft.Graph.Entra
+}
+  
+Describe "Set-EntraGroupLifecyclePolicy " {
```
Comment (KenitoInc) [inline test/module/Entra/Set-EntraGroupLifecyclePolicy.Tests.ps1:25]:
```suggestion
Describe "Set-EntraGroupLifecyclePolicy" {
```

File: test/module/Entra/Get-EntraAuthorizationPolicy.Tests.ps1 (hunk 1 above)
Comment (snehalkotwal) [reply test/module/Entra/Get-EntraAuthorizationPolicy.Tests.ps1:36]: @KenitoInc Id parameter is missing in mg graph so it is not working on the Entra cmdlet.

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**